libs/vscode/tasks/src/lib/cli-task-commands.ts

```diff
@@ -49,8 +47,18 @@ export function registerCliTaskCommands(
     );
   });
 
-  commands.registerCommand('nx.run', () =>
-    selectCliCommandAndPromptForFlags('run')
+  commands.registerCommand(
+    'nx.run',
+    (project?: string, target?: string, configuration?: string) => {
+      let flags;
+      if (configuration) {
```

Comment:
I went with `-c configuration` here because `ng run` on an angular cli project doesn't take --prod as an argument, but it does accept -c. Technically, the CodeLens is formatted as `nx|ng run project:target:configuration` which is likely the most preferred form, but I wanted to pass configuration as flags and run it in the same fashion as these other cli task commands.